index.bs

```diff
@@ -456,8 +458,10 @@ If we extend this concept to include those documents that are often too large to
 
 <div class="note">
     From this scraping, we can reason about several aspects:
-    * We use only some digital credentials to verify our identity (e.g., driver's license, passport), which have additional attributes that can be useful other than identification.
-    * Many other credentials are related to our features or entitlements (e.g., degree certificate, work permit), which allow us to do many things but not identify ourselves.
+    * All credentials describe some of our properties, but we can only use some of them to identify ourselves, and that depends on the specific context. 
+        For example, credentials like a passport is accepted everywhere (according to agreements between governments).
+    * Conversely, we can use a utility bill to prove our physical address, even though the bill was not created for that purpose. 
```

Comment:
```suggestion
    * The third is that credential reuse is often tied to the strength of its subproperties. For example, a utility bill may be used to prove our physical address in a KYC context (because it is tied to the mail system) but may not be sufficient in a KYC context to prove our name or surname.
```